.changeset/spotty-islands-study.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': major
```

Comment:
If I understand correctly, this is a bug fix and not a breaking change. If I'm wrong, what does this break?

Reply:
Yes, this is a bug fix. I have never did a PR like this in an open source project before. Sorry.